app/models/dog_searcher.rb

```diff
@@ -71,7 +71,9 @@ def self.search(params: {}, manager: false)
   private
 
   def tracking_id_search?
```

Comment:
Okay, looks like letters sometime show up in the microchip values as well.   Maybe something like this for a different approach?  https://stackoverflow.com/questions/9379884/search-multiple-columns-rails